# User Define Exception / Raising Exception
# 1) Raise Statement
# 2) Assert Statement

# 1) Raise Statement-------
# Syntax---->  raise exception_name[optional argument]



# a=int(input("enter any positive number:"))
# if a<0:
#     raise exception("not a negative number")
# else:
#     print("Great----you enter positive number")




# a = int(input("Enter any value between 5 and 9:"))
#
# if(a<5  or a>9):
#   raise  ValueError("Value should be between 5 and 9")




# 2) Assert Statement ----------------
# Syntax----->   assert exception[argument]

# print("use of assert statement")
# def negative_check(Number):
#     assert (Number>0),"oops negative number"
#     print(Number*n)
#
# n=int(input("enter number: "))
# negative_check(n)



